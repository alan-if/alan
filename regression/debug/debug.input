o
o 2
i


i a*
i *m
i *o*r
i 4
events
q
