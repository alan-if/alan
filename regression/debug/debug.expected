########## debug ##########

        No warnings or errors detected.
        9 informational message(s).

<Hmm, this is a little-endian machine, fixing byte ordering.... OK.>

adbg> o
Objects: 
    [6] door ("door"), at [5] Hall ("Hall") 
    [9] rats ("rats"), at [8] cellar ("cellar") 
    [11] tape ("spool of computer tape"), at [10] store ("store") 
    [13] book ("old book"), at [12] first_floor ("First Floor") 
    [15] ladder ("3 metre long ladder"), at [14] office ("office") 
    [18] terminal ("rather heavy computer terminal"), at [17]
terminal_room ("Terminal Room") 
    [19] coin ("small coin"), at [1] #nowhere ("#nowhere") 
    [24] nest ("birds nest"), at [23] walless ("Walless Room") 
    [25] keys ("set of rusty keys"), at [1] #nowhere ("#nowhere") 
    [28] clock ("clock"), at [27] clockroom ("Clock Room") 
    [31] drawer ("drawer"), at [30] deskroom ("Small Office") 
    [32] desk ("desk"), at [30] deskroom ("Small Office") 
    [33] manual ("dirty manual"), at [1] #nowhere ("#nowhere") 
    [35] computer ("computer"), at [34] computer_room ("Computer Room") 
    [37] machine ("vending machine"), at [36] cafe ("cafe") 
    [38] candy ("old mouldy candy bar"), at [1] #nowhere ("#nowhere")
adbg> o 2
Instance 2 is not an object
adbg> i
Instances: 
    [1] #nowhere ("#nowhere") 
    [2] pseudowords ("pseudowords") (container), at [1] #nowhere ("
#nowhere") 
    [3] nowhere ("nowhere") 
    [4] outside ("Outside The Tall Building") 
    [5] Hall ("Hall") 
    [6] door ("door"), at [5] Hall ("Hall") 
    [7] stairs ("Stairs") 
    [8] cellar ("cellar") 
    [9] rats ("rats"), at [8] cellar ("cellar") 
    [10] store ("store") 
    [11] tape ("spool of computer tape"), at [10] store ("store") 
    [12] first_floor ("First Floor") 
    [13] book ("old book"), at [12] first_floor ("First Floor") 
    [14] office ("office") 
    [15] ladder ("3 metre long ladder"), at [14] office ("office") 
    [16] second ("Second Floor") 
    [17] terminal_room ("Terminal Room") 
    [18] terminal ("rather heavy computer terminal"), at [17]
terminal_room ("Terminal Room") 
    [19] coin ("small coin"), at [1] #nowhere ("#nowhere") 
    [20] third ("Third Floor") 
    [21] fourth ("Fourth Floor") 
    [22] holeroom ("Windy Room") 
    [23] walless ("Walless Room") 
    [24] nest ("birds nest"), at [23] walless ("Walless Room") 
    [25] keys ("set of rusty keys"), at [1] #nowhere ("#nowhere") 
    [26] fifth ("Fifth Floor") 
    [27] clockroom ("Clock Room") 
    [28] clock ("clock"), at [27] clockroom ("Clock Room") 
    [29] corr ("Corridor") 
    [30] deskroom ("Small Office") 
    [31] drawer ("drawer"), at [30] deskroom ("Small Office") 
    [32] desk ("desk"), at [30] deskroom ("Small Office") 
    [33] manual ("dirty manual"), at [1] #nowhere ("#nowhere") 
    [34] computer_room ("Computer Room") 
    [35] computer ("computer"), at [34] computer_room ("Computer Room") 
    [36] cafe ("cafe") 
    [37] machine ("vending machine"), at [36] cafe ("cafe") 
    [38] candy ("old mouldy candy bar"), at [1] #nowhere ("#nowhere") 
    [39] hero ("hero") (container), at [4] outside ("Outside The Tall
Building") 
    [40] #literal ("#literal"), at [1] #nowhere ("#nowhere")
adbg> 
adbg> 
adbg> i a*
No instances matched the pattern.
adbg> i *m
Instances: 
    [17] terminal_room ("Terminal Room") 
    [22] holeroom ("Windy Room") 
    [27] clockroom ("Clock Room") 
    [30] deskroom ("Small Office") 
    [34] computer_room ("Computer Room")
adbg> i *o*r
Instances: 
    [6] door ("door"), at [5] Hall ("Hall") 
    [12] first_floor ("First Floor") 
    [29] corr ("Corridor") 
    [35] computer ("computer"), at [34] computer_room ("Computer Room")
adbg> i 4
The [4] outside ("Outside The Tall Building") Isa location[2] 
    Attributes: 
        visits[1] = 0
adbg> events
Events: 
    1 [terminal_response]: Not scheduled.
adbg> q

